packages/@sfcc-bff/productapi/package.json

```diff
@@ -15,7 +15,7 @@
   "dependencies": {
     "@sfcc-core/core": "1.0.0-alpha.1",
     "@sfcc-core/core-graphql": "1.0.0-alpha.1",
-    "commerce-sdk": "0.2.0-alpha.6",
+    "commerce-sdk": "1.3.0-alpha.8",
```

Comment:
This change is already checked in to integration branch.

Reply:
yes, this branch is not going to be check in to integration.